net-misc/asterisk-opus/asterisk-opus-13.0.1.3.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit multilib
+
+DESCRIPTION="OPUS codec and supporting files for asterisk"
+HOMEPAGE="https://wiki.asterisk.org/wiki/display/AST/Codec+Opus"
+
+AST_PV="$(ver_cut 1-2)"
+MY_PV="$(ver_rs 2 _)"
+
+SRC_URI="x86? (
+	http://downloads.digium.com/pub/telephony/codec_opus/asterisk-${AST_PV}/x86-32/codec_opus-${MY_PV}-x86_32.tar.gz
+)
+amd64? (
+	http://downloads.digium.com/pub/telephony/codec_opus/asterisk-${AST_PV}/x86-64/codec_opus-${MY_PV}-x86_64.tar.gz
+)"
```

Comment:
Sorry I failed to notice this the first time, but:
1: you could sort the USES, amd64 before x86.
2: use `https://` since it's available.